olive/passes/onnx/nvmo_quantization.py

```diff
@@ -0,0 +1,116 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+# --------------------------------------------------------------------------
+from copy import deepcopy
+from pathlib import Path
+from typing import Any, Callable, Dict, Union
+
+from olive.cache import get_local_path_from_root
+from olive.hardware.accelerator import AcceleratorSpec
+from olive.model import OliveModelHandler
+from olive.model.utils import resolve_onnx_path
+from olive.passes import Pass
+from olive.passes.onnx.common import model_proto_to_olive_model
+from olive.passes.pass_config import ParamCategory, PassConfigParam
+from olive.resource_path import OLIVE_RESOURCE_ANNOTATIONS
+from olive.strategy.search_parameter import Categorical
+
+# static quantization specific config
+_dataloader_config = {
+    "data_dir": PassConfigParam(
+        type_=OLIVE_RESOURCE_ANNOTATIONS,
+        category=ParamCategory.DATA,
+        description="""
+            Path to the directory containing the dataset.
+            For local data, it is required if quant_mode is 'static' and dataloader_func is provided.
```

Comment:
Just some minor comments about the description of the config parameters. Please remove the references to `quant_mode` and `static`. 

If a calibration data loader is always needed, please add `required=True` under `dataloader_func` and update the descriptions of the other related params to remove the "only used if ..." part.

Can you also add a docstring under pass class to give a description of what this pass does? Thanks!